classes/controller/AdminController.php

```diff
@@ -2054,8 +2054,6 @@ private function getNotificationTip($type)
     {
         $tips = [
             'order' => [
-                $this->trans('Did you check your conversion rate lately?', [], 'Admin.Navigation.Notification'),
```

Comment:
I don't get why you remove these translations
What was their usage ?

Reply:
They were randomly used on page load, if you feel like you can remove the "random thing"...

Reply:
Page are not migrated, afaik these strings are not used anymore :thinking:

Reply:
They are 100% used as I tested this, in the notification window, but maybe I had luck and launched on the same string on 4-5 refresh of the page?